streamer-ui/packages/server/routes/pdb.js

```diff
@@ -106,7 +106,7 @@ var _getProjectsV1 = function(req, res, next) {
         con.query(sql, function (err, results) {
             if (err) {
                 con.end();
-                return next(createError(500, err.messsage));
+                return next(createError(500, err.message));
```

Comment:
a bugfix?

Reply:
@yarikoptic thanks for the fix.  Indeed, this typo leads to broken code.